code/game/machinery/dance_machine.dm

```diff
@@ -392,7 +396,7 @@
 		for(var/i in 1 to speed)
 			M.setDir(pick(GLOB.cardinals))
 			for(var/mob/living/carbon/NS in rangers)
-				NS.set_resting(!NS.resting, TRUE, TRUE)
+				NS.set_resting(!NS.resting, TRUE)
```

Comment:
```suggestion
				NS.set_resting(!NS.resting, TRUE, TRUE)
```
This change does not seem intentional. Try updating your master to latest code before making new branches to avoid this kind of issue.

Reply:
I did? I don't know why it did that. I'll fix it back when I wake up for the day.

Reply:
There's no rush, this can't be merged in another 16 hours even, but you can simply accept the suggestion here.
This is a fix I did some time ago being reverted, so I assume it was a problem with conflict resolution or master not up-to-date.

Reply:
I'm honestly confused - I always pull from upstream/master before opening a new branch.